App.js

```diff
@@ -1,26 +1,48 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState, useEffect } from 'react';
+import 'react-native-gesture-handler'
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+import { OracleMessage } from './Components/OracleMessage';
+import { ShakeEventFunction } from './Components/ShakeEventFunction';
+import { StartScreen } from './Components/StartScreen';
+import { AboutApp } from './Components/AboutApp';
 
 const App = () => {
+  //state that stores the state of the initial screen
+  const [initialScreen, setInitialScreen] = useState(true) 
```

Comment:
It seems that you sometimes add semicolons, and sometimes don't - let's be consistent about that :)